test/conformance/ptp/ptp.go

```diff
@@ -126,10 +126,9 @@ var _ = Describe("[ptp]", func() {
 		var fifoPriorities map[string]int64
 
 		execute.BeforeAll(func() {
-			if isSingleNode {
-				Skip("Running in single node mode")
-			}
-			if !discovery.Enabled() {
+			// Configure PTP only if more that 1 node is available
+			// if only 1 node is present, only option is to discover config
+			if !discovery.Enabled() && !isSingleNode{
```

Comment:
Should we skip if discovery isn't enabled and it's a single node?

Reply:
I was assuming that discovery would be mandatory for single node. Will remove it.

Reply:
if discovery is not enabled, then you need to create default PTPconfig for a single node based on request either as OC or BC.